specs/gluster_cluster_list_view.adoc

```diff
@@ -0,0 +1,142 @@
+// vim: tw=79
+
+= Tendrl Definitions: Display list of clusters discovered by Tendrl
+
+Display list of Gluster clusters present in Tendrl. The list should have functionality for 
+
+* filtering by name.
+* sorting by name, status, cluster version and managed.
+
+Cluster list will be sorted by name by default.
+
+== Problem description
+
+Tendrl should show list of Gluster clusters present by Tendrl along with their details.
+
+The details shown for each cluster in UI is -
+
+* Cluster Name
+* Status
+* Managed State
+* Cluster Version
+* Import button(if ready to import)
+* Launch Dashboard button(if already imported)
+* Cluster Details button(if already imported)
+* Message to state cluster configurable state for eg. ready to import, cluster misconfigured.
+
+The workflow is as follows:
+
+* API will provide the list of clusters to the Tendrl.
+
+* User can click on expand icon of each cluster record to see the error messages/logs and hosts associated with it.
```

Comment:
@brainfunked @anivargi @nthomas-redhat What messages will be displayed in the "Message" tab if cluster is imported and ready to use?